runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -1156,6 +1156,18 @@ NNFW_STATUS nnfw_session::train_prepare(const nnfw_train_info *info)
 
   try
   {
+    for (uint32_t i = 0; i < getInputSize(); ++i)
+    {
+      auto info = _nnpkg->inputInfo(i);
+      onert::ir::Shape new_shape(info.shape());
+      for (int32_t i = 0; i < new_shape.rank(); i++)
```

Comment:
I think this for loop is unnecessary. And the local variable `i` is duplicated.

Reply:
Ah sorry it's my mistake. :(